artifacts/deployment/etcdproxy-controller.yaml

```diff
@@ -0,0 +1,118 @@
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: kube-apiserver-storage
+spec:
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: etcdproxy-sa
+  namespace: kube-apiserver-storage
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: etcdproxy-crd-clusterrole
+rules:
+- apiGroups: ["etcd.xmudrii.com"]
+  resources: ["etcdstorages"]
+  verbs: ["get", "watch", "list", "create", "update", "patch"]
```

Comment:
Question: @deads2k mentioned we should try to enforce that edit can be done only on Status. I was not able to find how we can accomplish this. Any ideas?

Reply:
status is a subresource. You write "etcdstorages/status". You also have to  enable status subresource in your CRD for that.